apps/src/lib/kits/maker/toolkit.js

```diff
@@ -169,7 +170,12 @@ function getBoard() {
     makerBoardAPI === MB_API ? MicroBitBoard : CircuitPlaygroundBoard;
 
   if (shouldRunWithFakeBoard()) {
-    return Promise.resolve(new FakeBoard());
+    // Check which maker is being enabled
+    if (makerBoardAPI === MB_API) {
+      return Promise.resolve(new FakeMBBoard());
+    } else {
```

Comment:
Do we need to account for a case where makerBoardAPI could be null or undefined?

Reply:
At this point `shouldRunWithFakeBoard()` returns `true` which means that `makerBoardAPI` will be assigned a value. The interesting thing is that if we are in a contained curriculum level with Circuit Playground enabled, `makerBoardAPI` is assigned the boolean `true`. If we are in regular curriculum level or standalone App Lab project. then the `makerBoardAPI` is assigned the string `"circuitPlayground"`. 
For any level/standalone project with micro:bit enabled, the `makerBoardAPI` value is assigned `"microbit"`. 
Thus, I think that I prefer keeping as is. I consulted with @sanchitmalhotra126 about this and he agrees.

Reply:
Adding a bit more context: the function `getBoard` is called in the function `connect` also defined in `toolkit.js`. The [first check](https://github.com/code-dot-org/code-dot-org/blob/13a3d37c7e159d6ed597788b6f87b50a81116b96/apps/src/lib/kits/maker/toolkit.js#L67) in `connect` is ensuring that the Maker Toolkit is enabled. The Maker Toolkit [is enabled](https://github.com/code-dot-org/code-dot-org/blob/13a3d37c7e159d6ed597788b6f87b50a81116b96/apps/src/applab/applab.js#L697) in `applab.js` if `config.level.makerlabEnabled` is assigned a value. This same value is assigned to `makerBoardAPI` in the `getBoard` function.

Reply:
👍

Reply:
Thanks for the details Alice! Might be good to just include a comment here that if shouldRunWithVirtualBoard() is true, then we can safely assume that MB_API should be set. If you'd like to include more context around the fact that MB_API may either be 'circuitPlayground' or true based on the value assigned in the level, then that might be helpful too, for posterity's sake